app/src/main/java/com/example/weather/screen/adapter/DailyAdapter.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.weather.screen.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.weather.data.model.entity.WeatherBasic
+import com.example.weather.databinding.LayoutItemDailyBinding
+import com.example.weather.utils.ext.getIcon
+import com.example.weather.utils.ext.kelvinToCelsius
+import com.example.weather.utils.ext.unixTimestampToDateString
+import com.example.weather.utils.ext.unixTimestampToHourString
+
+class DailyAdapter : RecyclerView.Adapter<DailyAdapter.DailyViewHolder>() {
+
+    private val listWeathers by lazy { mutableListOf<WeatherBasic>() }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): DailyViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        return DailyViewHolder(LayoutItemDailyBinding.inflate(inflater, parent, false))
+    }
+
+    override fun getItemCount(): Int {
+        return listWeathers.size
+    }
+
+    override fun onBindViewHolder(holder: DailyViewHolder, position: Int) {
+        holder.bindData(listWeathers[position])
+    }
+
+    fun updateData(listData: MutableList<WeatherBasic>) {
+        listWeathers.clear()
+        listWeathers.addAll(listData)
+        notifyDataSetChanged()
+    }
+
+    inner class DailyViewHolder(private val binding: LayoutItemDailyBinding) : RecyclerView.ViewHolder(binding.root) {
```

Comment:
```suggestion
    inner class ViewHolder(private val binding: LayoutItemDailyBinding) : RecyclerView.ViewHolder(binding.root) {
```
Nếu RecyclerView chỉ có 1 loại item layout thì để im là ViewHolder, nhiều loại khác nhau mới cần đặt tên